packages/ember-htmlbars/lib/helpers/component.js

```diff
@@ -0,0 +1,74 @@
+/**
+@module ember
+@submodule ember-htmlbars
+*/
+import { read } from "ember-metal/streams/utils";
+import { readComponentFactory } from "ember-views/streams/utils";
+import EmberError from "ember-metal/error";
+
+/**
+  `{{component}}` is a `Ember.Handlebars` helper for adding instances of
```

Comment:
`Ember.HTMLBars` or "template" helper?  Once 1.10 lands `Ember.Handlebars` is for backwards compat only (and Handlebars itself isn't actually used/needed)...